core/src/main/kotlin/org/evomaster/core/problem/security/data/ActionStubMapping.kt

```diff
@@ -0,0 +1,14 @@
+package org.evomaster.core.problem.security.data
+
+class ActionStubMapping (
+    val id: Long,
+    val actionName: String,
+    val stub: String,
+    val port: Int,
+    val url: String
+) {
+
+    fun getVerifierName(): String {
+        return "httpCallbackVerifier${id}"
```

Comment:
is this related to constant variable `httpCallbackVerifierName` ? if so, should reuse instead of duplicate the same string prefix